src/zui/ZUITimeline/TimelineUpdate.tsx

```diff
@@ -35,7 +37,10 @@ function isWildcardType<T extends ZetkinUpdate>(
   return false;
 }
 
-const TimelineUpdate: React.FunctionComponent<Props> = ({ update }) => {
+const TimelineUpdate: React.FunctionComponent<Props> = ({
+  onEditNote,
```

Comment:
This is a reasonable solution within the current framework to me, but I think we should be able to change this to be less "drilly" with the new object oriented model approach.